tests/cypress/e2e/components/identity.cy.ts

```diff
@@ -0,0 +1,51 @@
+import shortAddress from '@/utils/chainProperties'
+
+const users = [
+  {
+    address: 'FqCJeGcPidYSsvvmT17fHVaYdE2nXMYgPsBn3CP9gugvZR5',
+    name: 'deepologic',
+    twitter: '@deepologic',
+    startedMinting: 'Started minting over 1 year ago',
+  },
+  {
+    address: 'FUEGFWoPPtX4XkRzEnZnMgwGKENscHXAiEZy6kcgaDz1BY6',
+    name: 'Yumi Arts',
+    twitter: '@YumiArtsNFT',
+    startedMinting: 'Started minting 12 months ago',
+  },
+  {
+    address: 'FQBgFrkzevAZCvFZB485rQa6h7gsZQz3UXRuYwxAzG7zQ4w',
+    name: 'cwh',
+    twitter: '@ClownWorldHouse',
+    startedMinting: 'Started minting 10 months ago',
+  },
+]
+
+describe('Identity.vue component', () => {
+  users.forEach(({ address, name, twitter, startedMinting }) => {
+    it(`should get Identity stats for ${name}`, () => {
+      cy.visit(`/rmrk/u/${address}`)
+
+      cy.getCy('user-identity').should('contain.text', name)
+      cy.getCy('user-identity').find('a').realHover()
+      cy.get('.tippy-popper')
+        .should('exist')
+        .then(() => {
+          cy.getCy('identity-display').should('contain.text', name)
+          cy.getCy('identity-twitter').should('contain.text', twitter)
+          cy.getCy('identity-address').should(
+            'contain.text',
+            shortAddress(address)
+          )
+          cy.getCy('identity-started-minting').should(
+            'contain.text',
+            startedMinting
+          )
+          cy.getCy('identity-last-bought').should('exist')
+          cy.getCy('identity-collected').should('not.contain.text', 0)
+          cy.getCy('identity-created').should('not.contain.text', 0)
+          cy.getCy('identity-sold').should('not.contain.text', 0)
```

Comment:
What if the sold number is 10 or 20 with the zero 0. Could it work correctly? 👀

Reply:
oh, you're right, it will throw an error. it should be exact, not contains